src/components/dashboard/Claim.js

```diff
@@ -46,20 +57,34 @@ const Claim = props => {
   const [showDialog, hideDialog, showErrorDialog] = useDialog()
   const [loading, setLoading] = useState(false)
   const [claimInterval, setClaimInterval] = useState(null)
-  const [state, setState]: [ClaimState, Function] = useState({
+  const [pageState, setPageState]: [ClaimState, Function] = useState({
     nextClaim: '--:--:--',
     entitlement: (entitlement && entitlement.toNumber()) || 0,
     claimedToday: {
       people: '--',
       amount: '--',
     },
   })
+
+  // get the number of people who did claim today. Default - 0
+  const numberOfPeopleClaimedToday = get(pageState, 'claimedToday.amount', 0)
   const [queueStatus, setQueueStatus] = useState(undefined)
   const [showLoading, hideLoading] = useLoadingIndicator()
 
   const wrappedGoodWallet = wrapper(goodWallet, store)
   const advanceClaimsCounter = useClaimCounter()
 
+  // A function which will open 'learn more' page in a new tab
+  const openLearnMoreLink = useOnPress(() => openLink(Config.learnMoreEconomyUrl), [])
+
+  // format number of people who did claim today
+  /*eslint-disable */
+  const formattedNumberOfPeopleClaimedToday = useMemo(
+    () => numeral(numberOfPeopleClaimedToday).format('0a'),
```

Comment:
this formatting will be changed in the #1796 , right ?

Reply:
right, I mentioned it in previous comment